library.js

```diff
@@ -0,0 +1,111 @@
+let myLibrary = [];
+
+const info1 = document.getElementById("getBookForm");
```

Comment:
Optional: It will be better if there any meaningful name used for the variable. Like `const getBookForm= document.getElementById("getBookForm");`

Reply:
Same for others pointed bellow 👇